src/Goerli-DssSpell.t.base.sol

```diff
@@ -2328,6 +2337,31 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
         assertEq(getMat(_ilk), _endMat * RAY / 100);
     }
 
+    function checkIlkLerpIncreaseMatOffboarding(bytes32 _ilk, bytes32 _oldLerp, bytes32 _newLerp, uint256 _newEndMat) public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        LerpFactoryAbstract OLD_LERP_FAB = LerpFactoryAbstract(0xbBD821c291c492c40Db2577D9b6E5B1bdAEBD207);
+        LerpAbstract oldLerp = LerpAbstract(OLD_LERP_FAB.lerps(_oldLerp));
+
+        uint256 t = (block.timestamp - oldLerp.startTime()) * WAD / oldLerp.duration();
+        uint256 tickMat = oldLerp.end() * t / WAD + oldLerp.start() - oldLerp.start() * t / WAD;
+        assertEq(getMat(_ilk), tickMat);
+        assertEq(spotter.wards(address(oldLerp)), 0);
+
+        LerpAbstract newLerp = LerpAbstract(lerpFactory.lerps(_newLerp));
+
+        hevm.warp(block.timestamp + newLerp.duration() / 2);
+        assertEq(getMat(_ilk), tickMat);
+        newLerp.tick();
+        assertEqApprox(getMat(_ilk), (tickMat + _newEndMat * RAY / 100) / 2, RAY / 100);
+
+        hevm.warp(block.timestamp + newLerp.duration());
+        newLerp.tick();
+        assertEq(getMat(_ilk), _newEndMat * RAY / 100);
```

Comment:
Instead of this you could actually call the regular test.
So to be honest I would use the regular function + checking the old lerp is denied. That could be handled directly in the test and you avoid adding a new base function.